configuration/src/main/java/io/airlift/configuration/testing/ConfigAssertions.java

```diff
@@ -98,6 +101,14 @@ private ConfigAssertions()
             Object expectedAttributeValue = expectedAttributeValues.get(attribute.getName());
 
             assertEquals(expectedAttributeValue, actualAttributeValue, attribute.getName());
+
+            String property = attribute.getInjectionPoint().getProperty();
+            if (property != null && property.contains("experimental")) {
+                Object value = actualAttributeValue;
+                if (value != null && !FALSE.equals(value) && !Optional.empty().equals(value)) {
```

Comment:
This will prevent any numerical `experimental` properties (e.g companion configs).

Reply:
> e.g companion configs

I am aware. I don't see what's the drawback. Please elaborate.

Reply:
Example configs:
```
experimental.dynamic-filtering-max-per-driver-row-count
experimental.dynamic-filtering-max-per-driver-size
```
they have reasonable defaults.